test/contracts/NFTFloorOracle.t.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.10;
 
-import "../contracts/misc/NFTFloorOracle.sol";
-import "../foundry_libs/ds-test/src/test.sol";
+import "../../contracts/misc/NFTFloorOracle.sol";
+import "../../lib/ds-test/src/test.sol";
```

Comment:
Foundry has a default mapping for library. I have fixed it in `foundry` branch. You may want to take a look